### DISCUSSION TITLE
Denylist and LSPosed

### DISCUSSION THREAD

**Discussion by YeyeX64** [LSPosed/LSPosed#2731] [Q&A]
Hello,

When i have Denylist actived in Magisk (26.2 on android 13 MIUI 14), i can't launch LSPosed ?

Someone have a tip ?

Best regards

**Comment by mywalkb** [comment]
When denylist is active zygisk is not injected so also LSPosed is not injected.
You can implement a bypass root detect with LSPosed or you can use Shamiko or my module mywalkb/DenylistUnmount.
With Shamiko or my module denylist must be configured but not enabled.
In my module zygisk is inject but magisk in unmounted so simple detector doesn't detect root (su binary). 
Shamiko is better but is closed source.

**Comment by YeyeX64** [comment]
Very thanks. I try Shamiko, without denylist, seem's to work. Root not detected by apps. 
Thanks
